map_num_long: 2
map_num_wide: 2
non_fly_list: ['(0,1)','(1,1)']
map_pre_dict: {'(0,0)':['N'],'(1,0)':['N'],'(0,1)':['S'],'(1,1)':['S']}

